src/pages/access_review/AccessReviewHeader.js

```diff
@@ -55,7 +60,7 @@ export const AccessReviewHeader = hh(class AccessReviewHeader extends React.Pure
               div({ style: SMALL },
                 "Review the Application Summary and Data Use Limitations to determine if the researcher should be granted access to the data."
               ),
-              div({ style: SMALL }, "DAC Chairs can optionally vote as a member.")
+              div({ style: SMALL }, this.state.dacChairMessage )
```

Comment:
This should probably not be rendered at all if the prop isn't there:
```suggestion
              div({ isRendered: !fp.isNil(props.dacChairMessage), style: SMALL }, props.dacChairMessage)
```